tempesta_fw/http_msg.c

```diff
@@ -26,14 +26,40 @@
 #include "http_msg.h"
 #include "ss_skb.h"
 
+/**
+ * Build TfwStr repesented HTTP header.
+ * @name	- header name without ':';
+ * @value	- header value;
+ */
+TfwStr *
+tfw_http_msg_make_hdr(TfwPool *pool, const char *name, const char *val)
+{
+	size_t n_len = strlen(name);
+	size_t v_len = val ? strlen(val) : 0;
+	const TfwStr tmp_hdr = {
+		.ptr = (TfwStr []){
+			{ .ptr = (void *)name,	.len = n_len },
+			{ .ptr = S_DLM,		.len = SLEN(S_DLM) },
+			{ .ptr = (void *)val,	.len = v_len },
+		},
+		.len = n_len + SLEN(S_DLM) + v_len,
+		.eolen = 2,
+		.flags = (val ? 3 : 2) << TFW_STR_CN_SHIFT
+	};
+
+	return tfw_strdup(pool, &tmp_hdr);
+}
+EXPORT_SYMBOL(tfw_http_msg_make_hdr);
```

Comment:
There is no need to export the symbol. Please never export symbols in vain and for unit tests (I fixed several such exports)!